docs/reST/ref/image.rst

```diff
@@ -81,18 +82,19 @@ following formats.
 
    The returned Surface will contain the same color format, colorkey and alpha
    transparency as the file it came from. You will often want to call
-   ``Surface.convert()`` with no arguments, to create a copy that will draw
-   more quickly on the screen.
+   :func:`pygame.Surface.convert()` with no arguments, to create a copy that
+   will draw more quickly on the screen.
 
-   For alpha transparency, like in .png images, use the ``convert_alpha()``
-   method after loading so that the image has per pixel transparency.
+   For alpha transparency, like in .png images, use the
+   :func:`pygame.Surface.convert_alpha()` method after loading so that the
+   image has per pixel transparency.
 
-   pygame may not always be built to support all image formats. At minimum it
-   will support uncompressed ``BMP``. If ``pygame.image.get_extended()``
-   returns 'True', you should be able to load most images (including PNG, JPG
+   Pygame may not always be built to support all image formats. At minimum it
```

Comment:
Always, always use lower case pygame.

This section does seem to be a repeat. Remove the whole sentence "pygame may not always be built to support all image formats."?

Reply:
I think "the start of a sentence" > normal pygame casing rules.

It is a repeat, but it's relevant information for the function, and it links to another relevant function, so I'm inclined to leave it.